### PR TITLE
Cu 862j6m3h7 utworzenie szkieletu aplikacji i podpicie do routera s18288pejotedupl

### PR DESCRIPTION
**Pull request by s18288**

### CODE REVIEW

**Comment by s20146** [comment]
Task linked: [CU-862j6m3h7 Utworzenie szkieletu aplikacji i podpięcie do routera](https://app.clickup.com/t/862j6m3h7)

File: React/src/types/roleEnum.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
     '*' ='*',
-    a = 'a',
-    b = 'b',
     ADMIN = 'ADMIN',
-    CLOUD_ADMIN = 'CLOUD_ADMIN'
+    CLOUD_ADMIN = 'CLOUD_ADMIN',
+    MANAGER = 'MANAGER', 
+    SALES_REPRESENTATIVE = 'SALES_REPRESENTATIVE',
+    EXPERT = 'EXPERT',
+    WAREHOUSEMAN = 'WAREHOUSEMAN',
```
Comment (kurylek) [inline React/src/types/roleEnum.ts]: Should be 'WAREHOUSE_MAN'

File: React/src/types/roleEnum.ts [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+    WAREHOUSE_MANAGER = 'WAREHOUSE MANAGER',
```
Comment (kurylek) [inline React/src/types/roleEnum.ts]: Should be 'WAREHOUSE_MANAGER'

File: React/src/types/roleEnum.ts [hunk 3]
```diff
@@ -1,7 +1,12 @@
 export enum Role {
     '*' ='*',
-    a = 'a',
-    b = 'b',
     ADMIN = 'ADMIN',
-    CLOUD_ADMIN = 'CLOUD_ADMIN'
+    CLOUD_ADMIN = 'CLOUD_ADMIN',
+    MANAGER = 'MANAGER', 
+    SALES_REPRESENTATIVE = 'SALES_REPRESENTATIVE',
+    EXPERT = 'EXPERT',
```
Comment (kurylek) [inline React/src/types/roleEnum.ts]: Should be 'SPECIALIST'

File: React/src/utils/pageList.tsx [hunk 4]
```diff
@@ -49,82 +65,121 @@ export const pageList: Array<PageProps> =
                 [
                     {
                         inNav: true,
-                        name: 'Mock',
-                        path: '/mock',
```
Comment (kurylek) [inline React/src/utils/pageList.tsx:54]: Users were redirected to /mock after login, now they get 404 after login

**Review by kurylek — CHANGES_REQUESTED** [review]
Need some changes in role names, fix in redirect after login (since we don't use /mock right now), and there is 'allowedRoles: [Role["*"]]' on every page, so eg. fitterr can see everything.

**Review by kurylek — APPROVED** [review]
Would be nice if main page is first link in menu, but it works fine!

**Pull request merged**